2-mandatory/1-create-functions.js

```diff
@@ -76,12 +137,12 @@ test("sortArray function returns a sorted version of the array", () => {
   ]);
 });
 
-test("sortArray function doesn't change the passed in array", () => {
+/*test("sortArray function doesn't change the passed in array", () => {
   const before = ["a", "n", "c", "e", "z", "f"];
   const copy = before.slice();
   sortArray(before);
   expect(before).toEqual(copy);
-});
+});*/
```

Comment:
It looks like you've forgotten to uncomment the tests :)

Reply:
Thank you for spotting that. I will make sure I fix it. 👍